File: sdkwork-app-sdk-go/types/user_skill_vo.go
package types

// 用户技能安装信息
type UserSkillVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	UserSkillId int `json:"userSkillId"`
	SkillId int `json:"skillId"`
	Enabled bool `json:"enabled"`
	Config map[string]interface{} `json:"config"`
	InstalledAt string `json:"installedAt"`
	LastEnabledAt string `json:"lastEnabledAt"`
	LastUsedAt string `json:"lastUsedAt"`
	UsedCount int `json:"usedCount"`
	Skill SkillVO `json:"skill"`
}

File: sdkwork-app-sdk-csharp/Models/UserSkillVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class UserSkillVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? UserSkillId { get; set; }
        public int? SkillId { get; set; }
        public bool? Enabled { get; set; }
        public Dictionary<string, object>? Config { get; set; }
        public string? InstalledAt { get; set; }
        public string? LastEnabledAt { get; set; }
        public string? LastUsedAt { get; set; }
        public int? UsedCount { get; set; }
        public SkillVO? Skill { get; set; }
    }
}
